turbinia/processors/google_cloud.py

```diff
@@ -396,8 +399,8 @@ def PostprocessDetachDisk(evidence):
 
   config.LoadConfig()
   instance_name = GetLocalInstanceName()
-  project = GoogleCloudProject(project_id=config.PROJECT,
-                               default_zone=config.ZONE)
+  project = GoogleCloudProject(
+      project_id=config.PROJECT, default_zone=config.ZONE)
```

Comment:
I don't get this one either.  Everywhere else it seems that it likes to put the first parameter on the same line (like this was previously), and then put the second parameter on the second line even with it.  Why does it handle it differently here?